src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -549,6 +567,20 @@ static void WindowNewRideRefreshWidgetSizing(rct_window* w)
     int32_t width, height;
 
     // Show or hide unrelated widgets
+
+    if (_windowNewRideCurrentTab < WINDOW_NEW_RIDE_PAGE_SHOP)
+    {
+        window_new_ride_widgets[WIDX_GROUP_BY_TRACK_TYPE].type = WindowWidgetType::Checkbox;
+        window_new_ride_widgets[WIDX_RIDE_LIST].top = 62;
```

Comment:
It would be better if the shop page just had a grey bar with nothing on it, rather than hiding it. Now it just jumps up a bit, which is quite jarring.

Reply:
Yeah fair enough